addons/web_editor/static/src/components/media_dialog/video_selector.xml

```diff
@@ -3,27 +3,34 @@
 
 <t t-name="web_editor.VideoOption">
     <div class="mb-1">
-        <label class="o_switch" t-on-change="props.onChangeOption">
-            <input type="checkbox" t-att-checked="props.value"/><span/>
+        <label class="d-flex align-items-start gap-2 cursor-pointer" t-on-change="props.onChangeOption">
+            <div class="o_switch flex-shrink-0">
+                <input type="checkbox" t-att-checked="props.value"/>
+                <span/>
+            </div>
             <span t-esc="props.label"/>
-            <span t-if="props.description" class="small text-muted ms-auto" t-esc="props.description"/>
+            <span t-if="props.description" class="text-muted" t-esc="props.description"/>
         </label>
     </div>
 </t>
 
+<t t-name="web_editor.VideoIframe">
+    <iframe height="720" width="1280" class="o_video_dialog_iframe mw-100 mh-100 overflow-hidden shadow" allowfullscreen="allowfullscreen" frameborder="0" t-att-src="this.props.src"/>
```

Comment:
:socks: could be multi lined :)

Reply:
Is the fixed height / width ok even on small mobiles ?

Reply:
It seems like the `width` and `height` attribute are ignored. The video preview has an absolution position and fills the parent element. It's maybe an optimization: Setting those attributes will help the browser avoiding "layout shift" when loading the iframe.

Reply:
Actually the `width` and `height` attributes are overridden by the css style of the element, which specifies `100%`. Not sure about the layout shift thing, if you're convinced it is useful, we can keep the attributes, if not I'd remove them :)